src/main/java/nextstep/step2/domain/Ladder.java

```diff
@@ -0,0 +1,19 @@
+package nextstep.step2.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Ladder {
+
+    private final List<Line> lines = new ArrayList<>();
+
+    public Ladder(int height, int personCount) {
+        for (int i = 0; i < height; i++) {
+            lines.add(new Line(personCount));
+        }
+    }
```

Comment:
```suggestion
public class Ladder {

    private final List<Line> lines;

    public Ladder(List<Line> lines) {
        this.lines = lines;
    }

    public Ladder(int height, int personCount) {
        this(getLines(height, personCount));
    }
```

생성자를 다양하게 쓰기위해서 초기화를  주생성자로 이동시켜 볼것 같은데 어떻게 생각하시나요~

Reply:
알겠습니다.